
---
sensor_msgs/Image camera